ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -41,7 +41,7 @@ open class ContactView: UIControl {
 
         var width: CGFloat {
             var width = avatarSize.size.width
-            switch UIApplication.shared.preferredContentSizeCategory {
+            switch UIView().traitCollection.preferredContentSizeCategory {
```

Comment:
Using [UITraitCollection.current](https://developer.apple.com/documentation/uikit/uitraitcollection/3238080-current) seems like a better option here.

Your branch seems a bit outdated. We've already bumped the target deployment version to iOS13.

Reply:
I've aligned to main branch.